solutions/USACO dec20-chains-bronze.mdx

```diff
@@ -0,0 +1,62 @@
+---
+id: usaco-1060
+source: USACO Bronze 2020 December
+title: Daisy Chains
+author: Danh Ta Chi Thanh
+---
+
+[Official Analysis (Java)](http://www.usaco.org/current/data/sol_prob2_bronze_dec20.html)
+
+
+<LanguageSection>
+
+<CPPSection>
+
+## C++ Implementation
+
+```cpp
+#include <iostream>
+#include <algorithm>
+#include <cstdio>
+
+using namespace std;
+
+void solve(){
+	int n;
+	cin >> n;
+	int petals[n];
+	for (int i=0;i<n;++i){
+		cin >> petals[i];
+	}
+	int photos = 0;
+    for(int i = 0; i < n; i++) {
+      	for(int j = i; j < n; j++) {
+        	int totalPetals = 0;
+        	for(int k = i; k <= j; k++) {
+          		totalPetals += petals[k];
+        	}
+        	bool present = false;
+        	for(int k = i; k <= j; k++) {
+          		if(petals[k] * (j-i+1) == totalPetals) {
+            		present = true;
+          		}
+        	}
+        	if(present) {
+          		++photos;
+        	}
+      	}
+    }
+	cout << photos;
+}
+
+int main(){
+    cin.tie(0)->sync_with_stdio(0);

```

Comment:
I would prefer if you put ios_base::sync_with_stdio(0); cin.tie(0);, like is explained in the guide